engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -113,8 +112,10 @@ public ConfigurationImpl(ValidationProvider<?> provider) {
 	private ConfigurationImpl() {
 		this.validationBootstrapParameters = new ValidationBootstrapParameters();
 		TypeResolutionHelper typeResolutionHelper = new TypeResolutionHelper();
-		if ( isJavaFxInClasspath() ) {
-			validatedValueHandlers.add( new JavaFXPropertyValueUnwrapper( typeResolutionHelper ) );
+		ClassLoader classLoader = getClass().getClassLoader();
```

Comment:
I don't think the intention was to change the `isJavaFxInClasspath` in classpath. In my world it would e kept as is. Then, if it returns `true`, use the `LoadClass` privileged action (which is used under the hood for `isJavaFxInClasspath`) to load `JavaFXPropertyValueUnwrapper`.